src/axolotl/utils/schemas/config.py

```diff
@@ -563,6 +563,13 @@ class AxolotlInputConfig(
         },
     )
 
+    tiled_mlp_use_original_mlp: bool | None = Field(
```

Comment:
I see this defaults as `None`; are there any implications of not using the original MLP? I'm guessing that certain models' implementation could be sensitive to this?